### ISSUE TITLE
Custom file path for back-ups

### ISSUE THREAD

**Issue by dennissiemensma** [dennissiemensma/dsmr-reader#558]
@jeroenpeters1986 mentioned the nice idea of optionally having a custom filepath for back-ups. This allows users to have their back-ups on a different file system (such as an USB device).

**Comment by jeroenpeters1986** [comment]
Yes, USB drives and also mounted remote folders like OwnCloud or a network drive :-)
